src/libraries/System.Net.Security/tests/FunctionalTests/SslStreamConformanceTests.cs

```diff
@@ -0,0 +1,34 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.IO;
+using System.IO.Tests;
+using System.Security.Cryptography.X509Certificates;
+using System.Threading.Tasks;
+
+namespace System.Net.Security.Tests
+{
+    public sealed class SslStreamMemoryConformanceTests : ConnectedStreamConformanceTests
```

Comment:
Why is this called SslStream**Memory**ConformanceTests?

Reply:
It uses an in-memory channel.  I was thinking, given typical use cases, we'd want to also add a similar set over NetworkStream.